### ISSUE TITLE
FisherPedersenCeilingDiffuser* convection coefficients are based on zone outlet temp but use zone supply temp as the reference temperature

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6539]
Issue overview
--------------
Per the Engineering Reference, the CeilingDiffuser convection algorithm uses the zone outlet temperature.

> The ceiling diffuser algorithm is based on empirical correlations developed by Fisher and Pedersen (1997). The correlation was reformulated to use the room outlet temperature as the reference temperature.

**The zone-level CeilingDiffuser algorithm is correct - it uses the zone air temperature as the reference temperature.**

*The problem is with the surface-level FisherPedersenCeilingDiffuser\* algorithms. These are not used by default by the AdaptiveConvection option. They are only used if the user inputs specifies them at the surface-level or as an override to the defaults in AdaptiveConvection.*

The [current code](https://github.com/NREL/EnergyPlus/blob/501af898146625a9612782389de781aaf94b4139/src/EnergyPlus/ConvectionCoefficients.cc#L5534-L5579), sets the reference temperature to the zone supply air temperature.
```
	EvaluateIntHcModels(
		} else if ( SELECT_CASE_var == HcInt_FisherPedersenCeilDiffuserFloor ) {
				Surface( SurfNum ).TAirRef = ZoneSupplyAirTemp;
```

Looking back in the archive, there were two functions, one with the [original coefficients](https://github.com/NREL/EnergyPlusArchive/blame/8fd5ad8428e585860a8bd3ec5b18346dfd643f48/SourceCode/HeatBalanceConvectionCoeffs.f90#L2312) for supply air temperature, and one with the [modified coefficients](https://github.com/NREL/EnergyPlusArchive/blame/8fd5ad8428e585860a8bd3ec5b18346dfd643f48/SourceCode/HeatBalanceConvectionCoeffs.f90#L2219) for zone outlet temperature.
```
SUBROUTINE CalcCeilingDiffuserIntConvCoeff(ZoneNum)
          ! The correlations shown below differ from (and are less accurate than) those shown
          ! in the reference above (Fisher 1997).  They have been reformulated with an outlet
          ! temperature reference in order to accomodate the structure of the EnergyPlus code.

! CalcCeilingDiffuserInletCorr should replace CalcCeilingDiffuser (above), if ZoneTempPredictorCorrector can
! ever be made to work correctly with the inlet air temperature.
SUBROUTINE CalcCeilingDiffuserInletCorr(ZoneNum, SurfaceTemperatures)
```
And these two functions are still present, CalcCeilingDiffuserInletCorr is never used at the moment. It would be used for zone-level CeilingDiffuser with zone supply temp if we wanted that option to be available. It's not clear if that's ever been tested.

So, for the surface-level FisherPedersenCeilingDiffuser* algorithms do we want to 
1. Keep the current coefficients and change to zone mean air temperature as the reference temperature, or
2. Keep supply air temperature as the reference temperature and change to the coefficients.
3. Offer both options.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus pre-v8.9.0
 - Direct communication from LBNL

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added EnergyPlusDevSupport\DefectFiles
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
The zone-level CeilingDiffuser algorithm is correct - it uses the zone air temperature as the reference temperature. It's the surface-level options that are wrong. See updated title and description for this issue.

**Comment by RKStrand** [comment]
@mjwitte I have looked at this and I think I agree with Option 1 above.  The equations match the information in the Engineering Reference except that the wrong temperature is used.  I will work on this as soon as I get my issues with building EnergyPlus resolved.  I don't think there is a need to offer either Option 2 or 3--just confuses things.  Let me know if you disagree.

**Comment by mjwitte** [comment]
That's the safest solution. Some day it would be nice to add the supply temperature based correlation as an option and see what the impact is.

**Comment by mjwitte** [comment]
Closed via #6764
